README.md

```diff
@@ -141,6 +143,20 @@ A `CD` event communicates the outcome of an application deployment pipeline exec
 | &#x2011;&#x2011;run_status_details    | Any extra details about the status of the job run executing the deployment.                                                                                   |                                                           |                                                                |
 | &#x2011;&#x2011;run_start_time        | The start time of the job run in milliseconds since the epoch. (e.g. `1626804346019`)                                                                         |                                                           |                                                                |
 | &#x2011;&#x2011;run_end_time          | The end time of the job run in milliseconds since the epoch. (e.g. `1626804346019`)                                                                           |                                                           |                                                                |
+### Usage with Faros Community Edition
+
+When using Faros Community Edition, you can use the tool in exactly the same way as described above. Just include the `community_edition` flag. The Faros API key is not needed, since the tool will call your locally deployed Hasura to perform mutations derived from the events. See the [Faros Community Edition repo](https://github.com/faros-ai/faros-community-edition) for more details.
+
+```sh
+$ ./faros_event.sh CI \
```

Comment:
If I run the Faros Community Edition not on a localhost and if I specify `--url https://my-faros.com:9999` would it work?

Reply:
Done in a separate PR 👍

Reply:
let's put a `--`url` argument in this example with localhost + port where the Hasura is